static/app/views/settings/components/settingsNavItem.tsx

```diff
@@ -1,5 +1,5 @@
 import * as React from 'react';
-import {Link} from 'react-router';
+import {Link as RouterLink} from 'react-router';
```

Comment:
would it be possible to use our link here too?

Reply:
Would be nice but we use `activeClassName` and `onlyActiveOnIndex` which might be tricky to add to our link due to the way it works.

Reply:
Can always revisit